### ISSUE TITLE
Admin 5 Rating feature - Filter by current language

### ISSUE THREAD

**Issue by klein0r** [ioBroker/ioBroker.admin#773]
Currently, all ratings (in all languages) are displayed for an adapter.

```
{
    "rating": {},
    "comments": [
        {
            "ts": "2021-04-24T16:31:37.000Z",
            "comment": "Отлично работает!",
            "version": "5.0.7",
            "uuid": false,
            "rating": 0
        },
        {
            "ts": "2021-04-25T08:07:25.000Z",
            "comment": "For an alpha version, things are going well at first. Now I hope that all adapters will be quickly adapted to the new functionalities.",
            "version": "5.0.8",
            "uuid": false,
            "rating": 4
        }
    ],
    "adapter": "admin"
}
```

It would be great, if the comments are requested in the configured language (e.g. german) or browser language.

**Comment by Apollon77** [comment]
But from where do we get the information about the "language of the user entered comment"? This do not need to match tothe UI language. This will be difficult

**Comment by GermanBluefox** [comment]
Alexa skills showing all languages too.

**Comment by klein0r** [comment]
> But from where do we get the information about the "language of the user entered comment"?

Thats not that easy and maybe not 100% accurate, but I think most people will enter the comment in the configured language.

But: That was just an idea after checking the first reviews. Feel free to close this issue.

**Comment by GermanBluefox** [comment]
Implemented in 5.0.10

**Comment by klein0r** [comment]
Nice solution! Thank you 🥰
